ape_trezor/_cli.py

```diff
@@ -1,120 +1,152 @@
-@click.argument("alias")
-@click.option("--hdpath", default="m/44'/60'/0'/0")
-def add(hdpath, alias):
-    if alias in accounts.aliases:
-        notify("ERROR", f"Account with alias '{alias}' already exists")
+@cli.command("list")
+@ape_cli_context()
+def list(cli_ctx):
+    """List the Trezor accounts in your ape configuration"""
+
+    trezor_accounts = _get_trezor_accounts()
+
+    if len(trezor_accounts) == 0:
+        cli_ctx.logger.warning("No accounts found.")
         return
 
-    path = container.data_folder.joinpath(f"{alias}.json")
-    try:
-        client = get_default_client()
-    except Exception as e:
-        raise Abort("Trezor device not found. Please connect via USB and unlock!") from e
-
-    notify("INFO", "Please enter passphrase to allow address discovery.")
-
-    account_hdpath = None
-    index_offset = 0
-    while not account_hdpath:
-        options = []
-        for index in range(index_offset, index_offset + 10):
-            options.append(str(index))
-            address = ethereum.get_address(client, parse_hdpath(f"{hdpath}/{index}"))
-            click.echo(f"{address}: {index}")
-        options.append("n")
-        if index_offset > 0:
-            options.append("p")
-        account_choice = click.prompt(
-            "Please choose the address you would like to add, "
-            "or type 'n' for the next ten entries (or 'p' for the previous 10)",
-            type=click.Choice(options),
-        )
-        if account_choice == "n":
-            index_offset += 10
-        elif account_choice == "p":
-            index_offset -= 10
-        else:
-            account_hdpath = f"{hdpath}/{account_choice}"
-
-    address = ethereum.get_address(client, parse_hdpath(account_hdpath))
-    path.write_text(
-        json.dumps(
-            {
-                "address": address,
-                "hdpath": account_hdpath,
-            }
-        )
-    )
-
-    notify("SUCCESS", f"A new account '{address}' has been added with the id '{alias}'")
-
-
-@cli.command(
-    short_help="Remove an Trezor account from your Ape configuration. \
-    (The account will not be deleted from the Trezor hardware device)"
+    num_accounts = len(accounts)
+    header = f"Found {num_accounts} account"
+    header += "s:" if num_accounts > 1 else ":"
+    click.echo(header)
+
+    for account in trezor_accounts:
+        alias_display = f" (alias: '{account.alias}')" if account.alias else ""
+        hd_path_display = f" (hd-path: '{account.hdpath}')" if account.hdpath else ""
+        click.echo(f"  {account.address}{alias_display}{hd_path_display}")
+
+
+def _get_trezor_accounts() -> List[TrezorAccount]:
+    return [a for a in accounts if isinstance(a, TrezorAccount)]
+
+
+@cli.command()
+@ape_cli_context()
+@non_existing_alias_argument()
+@click.option(
+    "--hd-path",
+    help=("The Ethereum account derivation path prefix. " f"Defaults to {HDBasePath.DEFAULT}."),
```

Comment:
```suggestion
    help=(f"The Ethereum account derivation path prefix. Defaults to {HDBasePath.DEFAULT}."),
```

Just can combine these